dasar/html/list.md

```diff
@@ -33,7 +52,34 @@ Unordered list merupakan list yang tidak berurutan, sehingga setiap item pada li
 </ul>
 ```
 
-disana juga terdapat tag `li` yang digunakan untuk membuat list item, sehingga untuk setiap item pada list tersebut perlu kita bungkus dengan tag `li`.
+Disana juga terdapat tag `li` yang digunakan untuk membuat list item, sehingga untuk setiap item pada list tersebut perlu kita bungkus dengan tag `li`.
+
 
 Contoh tampilan Unordered List :  
 ![unordered-list](unorderedList.jpg)
+
+
+## 4. Praktek!
+
+Dari materi di atas sudah dijelaskan ringkasan mengenai list, untuk lebih memahami lagi, Ikuti intruksi dibawah ini:
+
+1. Buatlah Website sederhana untuk menampilkan menu Favorit Anda.
+
+2. List yang perlu ditampilkan adalah hobi, makanan, lagu, dan film. 
+
+3. Untuk list hobi dan makanan berikan tag ``<ol/>``
+
+4. Untuk list lagu dan film berikan tag ``<ul>``
+
+  Contoh tampilan hasil kasus diatas sebagai berikut: 
+
+
+  ![Latihan list](latihanList.jpg)
```

Comment:
yang bagian ini kyknya gak oke kalo langsung latihan, butuh contoh kasus dulu. Jadi untuk minggu2 ini fokus sampai contoh kasus dulu. Ntar kalau sudah rapi semua baru ditambahi latihan praktek

Soalnya kalo gak ada step by step contoh kasus, nanti pas miniclass fasilitator masih harus mendemokan contoh kasusnya, gak solving the problem silabus ini ntar